actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -252,6 +252,10 @@ def require(key)
     # Alias of #require.
     alias :required :require
 
+    def require_all(keys)
```

Comment:
If that's more preferable I can do that too. As its not important to me if its one method or two methods. I just want to have an ability to require multiple params in single call.

so let me know what is more preferable. incase case its `def require(*keys)`, how do you handle the return value with multiple fields?